### DISCUSSION TITLE
Bundle discovery with space in the root

### DISCUSSION THREAD

**Discussion by sirpi** [open-policy-agent/community#578] [OPA and Rego]
Hi everyone,

I noticed a strange behavior when the agent is configured with bundle discovery, and one of the bundles looks like this:

.manifest:
{ "revision": "abc", "roots": ["a/b/foo bar"] }

and the bundle contains the following file structure:

```
a
  b
    foo bar
      data.json
.manifest
.signatures.json
```

In this case the bundle is not loaded, but interestingly there is no error message either, only "Bundle loaded and activated successfully. Etag updated to ..."     

The same bundle loads perfectly if loaded statically:
opa run -s -b bundle.tar.gz

I am not sure whether this is intended or a bug, that's why I initiated a discussion.
And unfortunately we get the roots for these bundles from an external system, so we have no influence on them.
I can implement a workaround though by creating a "dictionary" bundle and using standard names in the roots, but if there would be a direct solution I would prefer that instead.

Any comment is appreciated.

**Comment by srenatus** [comment]
This sounds odd indeed. Can you please clarify what you've tried to come to this conclusion? ⏬ 
> In this case the bundle is not loaded,

**Reply by sirpi** [reply]
Fair enough. I called 
GET localhost:8182/v1/data
and the content of this bundle was missing.

**Reply by srenatus** [reply]
✔️ thanks for clarifying

**Reply by srenatus** [reply]
So this is definitely not something intentional. Feel free to file a bug, I suppose.

**Comment by ashutosh-narkar** [comment]
Closing this in favor of https://github.com/open-policy-agent/opa/issues/6704.
